WBooks/Models/WBooksUser.swift

```diff
@@ -32,6 +32,10 @@ internal struct WBooksUser {
         self.commentsCounter = commentsCounter
         self.unreadNotificationsCount = unreadNotificationsCount
     }
+    
+    func getFullName() -> String {
```

Comment:
For this sort of things where the performance cost is negligent I would use a computed property

```swift
var fullName: String {
  return "\(firstName) \(lastName)"
}
```

But never use computed property for things that require significant CPU use because one tends to assume that properties are "cheap"